recipe/meta.yaml

```diff
@@ -45,6 +45,11 @@ test:
   - bob.math
 
   commands:
+  # Use only 1 thread with OpenBLAS to avoid timeouts on CIs.
+  # This should have no other affect on the build. A user
+  # should still be able to set this (or not) to a different
+  # value at run-time to get the expected amount of parallelism.
+  - export OPENBLAS_NUM_THREADS=1
```

Comment:
Set the number of threads for OpenBLAS to 1. Seems to be timing out on CircleCI. As having parallelism is the only real change here, I expect this is likely the cause. Fixing it to 1 should solve this problem and let this proceed as before.